1.8/nexus/README.md

```diff
@@ -1,116 +1,92 @@
-# How to use Nexus 3 repository manager on DC/OS
+# How to use Nexus Repository Manager 3 on DC/OS
 
-[Nexus 3](http://www.sonatype.org/nexus/) is a repository manager that supports a broad variety of package managers, namely Bower, Docker, Maven 2, npm, NuGet, PyPI, and Raw site repositories. DC/OS allows you to quickly configure, install and manage a Nexus 3 instance.
+[Nexus Repository Manager 3](https://www.sonatype.com/nexus-repository-oss) is a repository manager that supports a broad variety of package managers, namely Bower, Docker, Maven 2, npm, NuGet, PyPI, and Raw site repositories. DC/OS allows you to quickly configure, install and manage a Nexus 3 instance.
 
 - Estimated time for completion: 10 minutes
 - Target audience: Anyone interested in running a repository manager.
 - Scope: Learn how to install Nexus 3 on DC/OS
 
+**Table of Contents**:
+- [Prerequisites](#prerequisites)
+- [Install Nexus 3](#install-nexus-3)
+  - [Ephemeral Configuration and Storage](#ephemeral-configuration-and-storage)
+  - [Persistent Local Storage](#persistent-local-storage)
+  - [Persistent External Storage](#persistent-external-storage)
+- [Using Nexus as a Docker Proxy](#using-nexus-as-a-docker-registry)
+- [Resources](#resources)
+
 ## Prerequisites
 
-- A running DC/OS 1.8 cluster with at least 1 node having at least 0.5 CPUs and 2 GB of RAM available.
-- [DC/OS CLI](https://dcos.io/docs/1.8/usage/cli/install/) installed.
+- A running DC/OS 1.8 cluster with at least 1 node having at least 1 CPUs and 2 GB of RAM available.
+- *Optional* [DC/OS CLI](https://dcos.io/docs/1.8/usage/cli/install/) installed.
 
 ## Install Nexus 3
 
-Assuming you have a DC/OS cluster up and running, we will discuss the installation of Nexus 3 in two variants for persistence. One will be with host volumes, and the other with external persisten volumes. The reason behind that is that the data of the repository / the artifacts need to persisted, so that the Nexus 3 application can be restarted without a data loss. 
-
-If you opt to use host volumes, make sure that there is a backup process in place, which will backup the data in the *data folder*, or that you use a shared filesystem mount (such as NFS) for it.
-
-### Host volumes
```

Comment:
> I second @tobilg and believe that it would be helpful to keep the existing cli, options.json... content.

Update to include options.json structure and fields with a couple examples.